samples/sensor/bmi270/src/main.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2020 Bosch Sensortec GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+#include <stdio.h>
+
+void main(void)
+{
+	const struct device *dev = device_get_binding(DT_LABEL(DT_INST(0,
+								       bosch_bmi270)));
+	struct sensor_value acc[3], gyr[3];
+	struct sensor_value full_scale, sampling_freq, oversampling;
+
+	printf("Device %p name is %s\n", dev, dev->name);
+
+	/* Setting scale in G, due to loss of precision if the SI unit m/s^2
+	 * is used
+	 */
+	full_scale.val1 = 2;            /* G */
+	full_scale.val2 = 0;
+	sampling_freq.val1 = 100;       /* Hz. Performance mode */
+	sampling_freq.val2 = 0;
+	oversampling.val1 = 1;          /* Normal mode */
+	oversampling.val2 = 0;
+
+	if (dev) {
```

Comment:
I suggest to check for the NULL in the line 18, print an error message and exit.
```c
        const struct device *dev;                                                             
        struct sensor_value acc[3], gyr[3];                                                   
        struct sensor_value full_scale, sampling_freq, oversampling;                          
                                                                                              
        dev = device_get_binding(DT_LABEL(DT_INST(0, bosch_bmi270)));                         
        if (dev == NULL) {                                                                    
                printf("Could not get %s device\n",                                                  
                       DT_LABEL(DT_INST(0, bosch_bmi270)));                                   
                return;                                                                       
        } 
```